content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,197 @@
+
+
+![Git Blame](https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.0/git_blame.png "Git Blame")
+
+<Badge text="Code navigation" color="lemon" size="small" onClick={() => window.open("/code-search, "_self")} />
+
+#### Quickly access answers within your codebase with a revamped reference panel 
+We've enhanced the code navigation experience with a new version of the reference panel. Updates include a new preview panel, the conslidation of definitions and references in the same tab, and allows for easier navigation through filters.
+
+![Reference panel revamp](https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.0/Reference%20Panel%20Image.png "Refernce panel revamp")
+
+[Docs](https://docs.sourcegraph.com/code_navigation/explanations/features#find-references)
+
+
+### 🏗️ High-leverage ways to improve your entire codebase
+
+<Badge text="Batch Changes" color="blue" size="small" onClick={() => window.open("/batch-changes", "_self")} />
+
+#### Make changes across all of your codebases at enterprise scale with server-side Batch Changes (beta)
+Batch Changes allows you to programmatically define changes across the codebase with a single declarative file. Now, changes can be run across thousands of repositories with a better development experience. Previously, you had to run src-cli locally to create batch changes, which could take an impractical amount of time or be brittle for large or resource-intensive batch changes. By running Batch Changes server-side you can:
+
+- Run large-scale or resource-intensive batch changes without clogging your local machine
+- Run large batch changes quickly by distributing them across an autoscaled pool of compute instances
+- Get a better debugging experience with the streaming of logs directly into Sourcegraph.
+
+This feature requires admins to set up executors (much like CI agents), which Sourcegraph will use to offload expensive tasks. Executors can also be used to run [Code Intelligence auto-indexing](https://github.com/sourcegraph/about/blob/bf341aafde75450c295aaec064470017020cdcd8/content/blogposts/2022/sourcegraph-4.0.md#TODO-link-to-section-or-docs) as well. One or more executors are available by default on Cloud instances of Sourcegraph.
+
+<Video 
+  source={{
+    webm: 'blog/release-post/4.0/Batch_Changes_Side_by_Side',
+    mp4: 'blog/release-post/4.0/Batch_Changes_Side_by_Side'
+  }}
+  loop={true}
+  title="Server-side Batch Changes"
+  caption="Side by side comparison of deploying Batch Changes locally vs. server-side"
+/>
+
+[Docs](https://docs.sourcegraph.com/batch_changes/explanations/server_side)
+
+
+### ☁️ Dedicated Sourcegraph Cloud instances for enterprise
+
+<Badge text="Admin" color="violet" size="small" onClick={() => window.open("URL_GOES_HERE", "_self")} />
+
+#### Get started with Sourcegraph faster than ever
```

Comment:
@lcolston08 @iskyOS - We can mention speed, but we should focus on language that highlights the absolute benefits of single-tenant Cloud: dedicated, isolated, secure, etc.

https://handbook.sourcegraph.com/company-info-and-process/communication/content_guidelines/style_and_mechanics/#sourcegraph-cloud